src/common/components/signInForm/signInForm.tpl.html

```diff
@@ -44,6 +44,6 @@ <h4 translate>Don&rsquo;t have an account?</h4>
   </div>
   <div class="need-help--content">
     <p><a ng-href="{{$ctrl.getOktaUrl()}}/signin/forgot-password" target="_blank" translate>Forgot your password?</a></p>
-    <p>Call (888) 278-7233 opt. 1, then 1.<br />9:00 a.m. to 5:00 p.m. ET, Monday-Friday,<br /> or email us at <a href="mailto:eGift@cru.org">eGift@cru.org</a>.</p>
+    <p>Call <a href="tel:+18882787233">(888) 278-7233</a> opt. 1, then 1.<br />9:00 a.m. to 5:00 p.m. ET, Monday-Friday,<br /> or email us at <a href="mailto:eGift@cru.org">eGift@cru.org</a>.</p>
```

Comment:
They asked me to remove the link for the phone number a few weeks ago, have they asked for it to be added back again?